alf/examples/icm_super_mario.gin

```diff
@@ -37,8 +37,7 @@ value/ValueNetwork.fc_layer_params=(256,256)
 value/ValueNetwork.conv_layer_params=%CONV_LAYER_PARAMS
 value/ValueNetwork.activation=@torch.nn.functional.elu
 
-ac/Adam.lr=3e-5
-ac/Adam.eps=1e-7
+ac/AdamTF.lr=3e-5
```

Comment:
Can possibly move the changes in ```icm_super_mario.gin``` to another PR as they are less relevant to the numerical gain calculation.